.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DGX3 Dev Container",
+    "dockerFile": "Dockerfile",
+    "runArgs": [
+        "--network",
+        "host",
+        "--gpus",
+        "all"
+    ],
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "terminal.integrated.shell.linux": "/bin/bash"
+            },
+            "mounts": [
+                "source=/home/fhgiais/.bashrc,target=/root/.bashrc,type=bind"
```

Comment:
This path looks pretty specific to our infrastructure. Does it make sense to check that in?